### PR TITLE
Grunt in container

### PR DESCRIPTION
**Pull request by wpears**
The grunt build now runs when building the container, meaning running containers is very light, won't differ across differs slaves, and, most importantly, doesn't require node and build tools for jenkins.

A casualty of this PR is the imagemin task, which alpine refused to build the native dependencies of, even given much urging. I was going to copy over the source images with the minified ones and check them in, but the savings for all affected images only totals ~7kb. So probably not worth the maintenance/slightly reduced quality at this juncture.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -289,17 +289,16 @@ module.exports = function (grunt) {
...
-        files: [{
-          expand: true,
-          cwd: '<%= yeoman.app %>/images',
-          src: '{,*/}*.{png,jpg,jpeg,gif}',
-          dest: '<%= yeoman.dist %>/images'
-        }]
-      }
-    },
-
+//    imagemin: {
```
Comment (hkeeler) [inline Gruntfile.js]: Let's either remove this all together, or comment _why_ its commented out.  This was blowing up due to some lib not being available in Alpine?

**Comment by hkeeler** [comment]
All reviewed up.  Looks good overall.  Just the one thing.

**Comment by awolfe76** [comment]
:+1: I like this way of handling it.

Something like https://www.npmjs.com/package/load-grunt-config might help that Gruntfile too. :wink:

**Comment by wpears** [comment]
Should be good to go now; pretty funny bug that would make tests fail in October only.

**Comment by hkeeler** [comment]
Looks good.  Merging...

**Pull request merged**